server.py

```diff
@@ -10,7 +10,9 @@
 
 buffers = {}
 
-app = Flask(__name__)
+app = Flask(__name__, static_url_path='', static_folder='web/dist')
+app.config['DEBUG'] = True
+app.config['TEMPLATES_AUTO_RELOAD'] = True
```

Comment:
These should not be set for deployment (release). I have set them further down, based on cmd args. I'll fix.